engine-nextgen/src/main/kotlin/graphql/nadel/validation/NadelSchemaValidationError.kt

```diff
@@ -226,32 +243,14 @@ sealed interface NadelSchemaValidationError {
         val parentType: NadelServiceSchemaElement,
         val overallField: GraphQLFieldDefinition,
         val hydration: UnderlyingServiceHydration,
-        val actorServiceQueryType: GraphQLObjectType,
     ) : NadelSchemaValidationError {
         val service: Service get() = parentType.service
 
         override val message = run {
             val of = makeFieldCoordinates(parentType.overall.name, overallField.name)
             val s = hydration.serviceName
-            val af = "${actorServiceQueryType.name}.${hydration.pathToActorField.joinToString(separator = ".")}"
-            "Field $of tried to hydrate from non-existent field $af on service $s"
-        }
-
-        override val subject = overallField
-    }
-
-    data class MissingHydrationActorFieldInOverallSchema(
```

Comment:
I don't think we actually reference these explicitly, so I don't think removing the type matters.